registry/Scripts/self-signed.ps1

```diff
@@ -1,13 +1,27 @@
 #requires -RunAsAdministrator
 
-# Password and Common Name of the certificate
-$PASSWORD=""
-$CN=""
-# Export path of the certificate
-$PATH=""
+Param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "Common DNS FQDN name to be used .")]
+    [string] $DNSCName,
+    [Parameter(Mandatory = $true, HelpMessage = "Certificate Password to be used to export the cert.")]
```

Comment:
`Certificate password` is probably enough

Reply:
Adding Password is cribbing and reporting as error. I will explain tomm in person.